utils/oomparser/oomparser.go

```diff
@@ -144,17 +144,18 @@ func (self *OomParser) StreamOoms(outStream chan *OomInstance) {
 			oomCurrentInstance := &OomInstance{
 				ContainerName: "/",
 			}
-			finished := false
-			for !finished {
+			for line := range lineChannel {
```

Comment:
@timstclair  The desired information is never in the start line so no problem.   Good you brought it up though.